pkg/console/subresource/deployment/deployment.go

```diff
@@ -238,15 +238,16 @@ func consoleVolumeMounts(vc []volumeConfig) []corev1.VolumeMount {
 
 func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
 	flag := ""
-	// Since the console-operator logging has different logging levels then the capnslog,
-	// that we use for console server(bridge) we need to map them to each other
+	// console server(bridge) expects flag log-level
```

Comment:
This comment can go away, it will no longer expect `log-level` after these merge.